ansible/roles/prosody/files/mod_muc_get_media_type.lua

```diff
@@ -0,0 +1,74 @@
+---
+--- Generated by Luanalysis
+--- Created by ltorje.
+--- DateTime: 06.02.2024 16:43
+---
+--- no regex, less logging, check room first and exist if set already
+--- getting the room --> e.g. <iq xmlns="jabber:client" id="26bbe412-6dde-4225-8ca8-0690a462b89d:sendIQ" to="borisyana1to1@conference.meet.jit.si/focus"
+--- room = borisyana1to1@conference.meet.jit.si
+
+local json = require "cjson";
+local jid = require "util.jid";
+local get_room_from_jid = module:require "util".get_room_from_jid;
+
+function handle_media_event(event)
+    local stanza = event.stanza;
+    if stanza.name == "iq" then
+        local room_jid = jid.bare(stanza.attr.to);
+        local room = get_room_from_jid(room_jid);
+
+        if room == nil then
+            return
+        end
+
+        if room.had_video then
```

Comment:
Do you want to add here ` or room.has_desktop`?

Reply:
Yeah, I mean, if you have already detected video or desktop short circuit it early.

Reply:
aaaa I see, you handled that. 👍